helion/__init__.py

```diff
@@ -22,7 +21,6 @@
     "Settings",
     "cdiv",
     "exc",
-    "jit",
```

Comment:
This will break BC.  Is there any harm in keeping it?  Seems like a common typo (since other langs like triton/jax use jit).

(We should change all the usages though.)